torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -1083,6 +1084,10 @@ std::shared_ptr<SugaredValue> toSugaredValue(
       auto memory_format = reinterpret_cast<THPMemoryFormat*>(obj.ptr());
       const auto v = static_cast<int64_t>(memory_format->memory_format);
       return toSimple(g.insertConstant(v, loc));
+    } else if (THPLinalgBackend_Check(obj.ptr())) {
```

Comment:
Is this necessary for jit support? I don't think previous switches added code here